static/js/zamboni/buttons.js

```diff
@@ -186,6 +186,9 @@ var installButton = function() {
         var opts = $.extend({addWarning: true}, options);
             warn = opts.addWarning ? addWarning : _.identity;
 
+        $('#downloadAnyway').attr('href',escape_($button.filter(':visible').attr('href')));
```

Comment:
whitespace after comma? I know this is just copied but you touched it so you're responsible 😇